libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -334,10 +340,18 @@ protected void next(String email) {
         }
 
         if (isValidEmail(email)) {
+            mIsValidEmail = true;
             startProgress();
             mRequestedEmail = email;
             mDispatcher.dispatch(AccountActionBuilder.newIsAvailableEmailAction(email));
         } else {
+            mIsValidEmail = false;
+            showEmailError(R.string.email_invalid);
+        }
+    }
+
+    protected void validEmail() {
```

Comment:
We could rename this method to something more meaningful like `showErrorIfEmailInvalid()`. Also this method could be made `private`. What do you think? 😄

Reply:
Totally agree, done :D

Reply:
Awesome. I made a comment about this!